### PR TITLE
master install:269 add a check for .ssh folder

### PR DESCRIPTION
**Pull request by daniwe4**
https://github.com/conceptsandtraining/doil/issues/269

### CODE REVIEW

File: setup/install.sh [hunk 1]
```diff
@@ -122,6 +122,15 @@ then
   exit
 fi
 
+# check for .ssh folder
+doil_check_ssh
+if [[ $? -ne 0 ]]
+then
+  doil_status_failed
+  doil_status_send_error "REQUIREMENT ERROR" "Missing .ssh folder!\n\tThis folder is required by doil.\n\tIf you only use public github repositories please create\n\tan empty .ssh folder in your home directory to fit this doil condition.\n\tOtherwise you should create ssh keys and link them with your private git repositories.\n\tFollow this link https://docs.github.com/de/authentication/connecting-to-github-with-ssh/about-ssh"
```
Comment (tbongers-cat) [inline setup/install.sh:130]: Maybe add a comma after "public github repositories" for better readability.
The last sentence could be "Follow this link ... for more information." to make it a bit clearer.

**Review by tbongers-cat — COMMENTED** [review]
Looks good, only two small text changes for better readability

File: setup/install.sh (hunk 1 above)
Comment (daniwe4) [reply setup/install.sh:130]: Thats right, thank you for this hints.

Comment (daniwe4) [reply setup/install.sh]: Done!

**Review by tbongers-cat — APPROVED** [review]
LGTM

**Pull request merged**